exchange/exchangetest/multi-bid-default-bid-limit-min.json

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
Because of how we updated the naming of `DefaultBidLimitMin` --> `DefaultBidLimit`

Could you remove the -min from the .json filename?

Reply:
done